linkerd/app/core/src/metrics.rs

```diff
@@ -96,11 +96,11 @@ pub struct RouteAuthzLabels {
     pub authz: Arc<policy::Meta>,
 }
 
-// Labels referencing an inbound HTTP local rate limit.
+/// Labels referencing an inbound HTTP local rate limit.
 #[derive(Clone, Debug, Eq, PartialEq, Hash)]
 pub struct HTTPLocalRateLimitLabels {
     pub server: ServerLabel,
```

Comment:
i noticed that we do not use this field in the `FmtLabels` implementation. should we? is this extraneous?

Reply:
It is used [here](https://github.com/linkerd/linkerd2-proxy/pull/3335/files#diff-cbb69683b3eaa8be3d3ab859c0b39bb83708d56eead603980bff21137de4591eR396) :-)

Reply:
oh, so it is!! thanks :slightly_smiling_face: